docs/my-first-function.md

```diff
@@ -192,6 +192,9 @@ $ PYTHONPATH=../../sdk/python python3 mesapy_echo.py mesapy_echo_payload.py 'Hel
 ## Simulation Mode
 
 Clone and build Teaclave (with the `-DSGX_SIM_MODE=ON` option in `cmake`).
+Note that if you are using Docker for Mac,
+[increase the size of allocated memory](https://docs.docker.com/docker-for-mac/) to
+avoid compilation error caused by out-of-memory.
 
```

Comment:
Should we mention the symptom (e.g. Issue 360) here so that the users know if they have increased the memory to large enough?

Reply:
Updated.